tests/unit/ActionRequired.spec.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint max-len: 0 */
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import { mount } from '@vue/test-utils'
+import ActionRequired from '@/components/Dashboard/ActionRequired.vue'
+import { ContactInfo } from '@/components/common'
+
+Vue.use(Vuetify)
+
+const vuetify = new Vuetify({})
+
+describe('ActionRequired', () => {
+  it('Displays ActionRequired', () => {
+    const wrapper = mount(ActionRequired, {
+      vuetify
+
+    })
+
+    // verify content
+    expect(wrapper.find('h3').text()).toBe('Action Required')
+    const paragraphs = wrapper.findAll('p')
+    expect(paragraphs.length).toBe(1)
+    expect(paragraphs.at(0).text()).toContain('BC Registries is missing information about your business')
+
+    wrapper.destroy()
+  })
+
+  it('Displays contact information on button click', async () => {
+    const wrapper = mount(ActionRequired, {
+      vuetify
+
+    })
+    const paragraphs1 = wrapper.findAll('p')
+    expect(paragraphs1.length).toBe(1)
+    expect(wrapper.find(ContactInfo).exists()).toBe(false)
+    // click the view button
+    wrapper.find('.details-btn').trigger('click')
+    await Vue.nextTick()
```

Comment:
Check if you can await the trigger function on the previous line instead. (Cleaner code.)

Reply:
I made the change but it's showing some warning in my code base,
So i think its better to wait for nextTick

![Screen Shot 2022-05-26 at 2 21 24 PM](https://user-images.githubusercontent.com/55506249/170582141-5871a856-112f-48d4-952b-f1afb674161c.png)

Reply:
Weird. I don't get that warning and my VS Code shows that `trigger` can return a promise.

![image](https://user-images.githubusercontent.com/10002889/170585664-b8ddcb31-0e2e-43d1-8a80-d398a447413b.png)

Anyway, it's fine.